packages/calypso-products/src/plans-list.tsx

```diff
@@ -657,56 +567,27 @@ const getPlanPremiumDetails = (): IncompleteWPcomPlan => ( {
 				' Google Analytics support,' +
 				' and the ability to monetize your site with ads.'
 		),
-	getPlanCompareFeatures: ( experiment, { isLoggedInMonthlyPricing } = {} ) =>
-		compact(
-			experiment
-				? [
-						// pay attention to ordering, shared features should align on /plan page
-						FEATURE_CUSTOM_DOMAIN,
-						FEATURE_MANAGED_HOSTING,
-						FEATURE_UNMETERED_BANDWIDTH,
-						FEATURE_UNLIMITED_TRAFFIC,
-						FEATURE_UNLIMITED_USERS_V2,
-						FEATURE_EMAIL_SUPPORT_V2,
-						FEATURE_FREE_THEMES_V2,
-						FEATURE_COLLECT_PAYMENTS_V3,
-						FEATURE_BLANK,
-
-						FEATURE_13GB_STORAGE,
-						FEATURE_ADVANCED_DESIGN_V2,
-						FEATURE_SEO_PREVIEW_TOOLS_V2,
-						FEATURE_REPUBLICIZE_V2,
-						FEATURE_PAID_NEWSLETTER,
-						FEATURE_BASIC_BACKUPS,
-						FEATURE_BLANK,
-
-						FEATURE_VIDEO_UPLOADS_V2,
-						FEATURE_NO_ADS,
-						FEATURE_WORDADS_INSTANT_V2,
-						isEnabled( 'themes/premium' ) ? FEATURE_PREMIUM_THEMES_V2 : null,
-						FEATURE_LIVE_CHAT_SUPPORT_BUSINESS_DAYS_V2,
-						FEATURE_BLANK,
-				  ]
-				: [
-						// pay attention to ordering, shared features should align on /plan page
-						FEATURE_CUSTOM_DOMAIN,
-						isLoggedInMonthlyPricing && FEATURE_LIVE_CHAT_SUPPORT_BUSINESS_DAYS,
-						isLoggedInMonthlyPricing && FEATURE_EMAIL_SUPPORT,
-						FEATURE_HOSTING,
-						FEATURE_JETPACK_ESSENTIAL,
-						! isLoggedInMonthlyPricing && FEATURE_EMAIL_LIVE_CHAT_SUPPORT_BUSINESS_DAYS,
-						isEnabled( 'themes/premium' ) ? FEATURE_PREMIUM_THEMES : null,
-						FEATURE_ADVANCED_DESIGN,
-						FEATURE_13GB_STORAGE,
-						FEATURE_NO_ADS,
-						FEATURE_MEMBERSHIPS,
-						FEATURE_PREMIUM_CONTENT_BLOCK,
-						FEATURE_SIMPLE_PAYMENTS,
-						FEATURE_REPUBLICIZE,
-						FEATURE_WORDADS_INSTANT,
-						FEATURE_VIDEO_UPLOADS,
-				  ]
-		),
+	getPlanCompareFeatures: ( _, { isLoggedInMonthlyPricing } = {} ) =>
+		compact( [
+			// pay attention to ordering, shared features should align on /plan page
+			FEATURE_CUSTOM_DOMAIN,
+			isLoggedInMonthlyPricing && FEATURE_LIVE_CHAT_SUPPORT_BUSINESS_DAYS,
+			isLoggedInMonthlyPricing && FEATURE_EMAIL_SUPPORT,
+			FEATURE_HOSTING,
+			FEATURE_JETPACK_ESSENTIAL,
+			! isLoggedInMonthlyPricing && FEATURE_EMAIL_LIVE_CHAT_SUPPORT_BUSINESS_DAYS,
+			isEnabled( 'themes/premium' ) ? FEATURE_PREMIUM_THEMES : null,
+			FEATURE_ADVANCED_DESIGN,
+			FEATURE_13GB_STORAGE,
+			FEATURE_NO_ADS,
+			FEATURE_MEMBERSHIPS,
+			FEATURE_PREMIUM_CONTENT_BLOCK,
+			FEATURE_SIMPLE_PAYMENTS,
+			FEATURE_GOOGLE_ANALYTICS,
```

Comment:
I noticed that `FEATURE_GOOGLE_ANALYTICS` is restored compared to production /plans page. 
I suppose this is now correct.